specs2/specs2_junit.bzl

```diff
@@ -0,0 +1,16 @@
+load("//specs2:specs2.bzl", "specs2_repositories", "specs2_dependencies", "specs2_version")
+load("//junit:junit.bzl", "junit_repositories")
+  
+def specs2_junit_repositories():
+  specs2_repositories()  
+  junit_repositories()
+  # Aditional dependencies for specs2 junit runner
+  native.maven_jar(
+      name = "io_bazel_rules_scala_org_specs2_specs2_junit_2_11",
+      artifact = "org.specs2:specs2-junit_2.11:" + specs2_version(),
```

Comment:
I don't see how that function helps (specs2_version) since only one version will have the right sha1 below, no (unless someone generates a sha1 collision for specs2! ;)